zerver/webhooks/stripe/view.py

```diff
@@ -14,26 +14,31 @@
     UnexpectedWebhookEventType
 from zerver.models import UserProfile
 
+
 class SuppressedEvent(Exception):
     pass
 
+
 class NotImplementedEventType(SuppressedEvent):
     pass
 
+
 @api_key_only_webhook_view('Stripe')
 @has_request_variables
 def api_stripe_webhook(request: HttpRequest, user_profile: UserProfile,
-                       payload: Dict[str, Any]=REQ(argument_type='body'),
-                       stream: str=REQ(default='test')) -> HttpResponse:
+                       payload: Dict[str, Any] = REQ(argument_type = 'body'),
+                       stream: str = REQ(default = 'test')) -> HttpResponse:
```

Comment:
We also don't have spaces around the `=` character in function arguments, which is a well-known convention in the Python world. I'd recommend checking out the [PEP8 document](https://www.python.org/dev/peps/pep-0008/) to see some of the well-known coding standards the Python community has! :)

One general piece of advice I have is that even if you feel like some code can be styled better, it is better to include them in a separate commit, not as part of a commit that is supposed to add a new feature(s) to something!